src/App/Services/Hosted/PrometheusExporterHostedService.cs

```diff
@@ -105,31 +105,29 @@ internal async Task RunExportersAsync()
                 }
 
                 await Task.WhenAll(tasks);
-                IsSuccessful.Set(1);
                 TotalSuccessfulScrapeActivations.Inc();
+                successfulRun = 1;
             }
             catch (AggregateException ae)
             {
                 foreach (var innerException in ae.Flatten().InnerExceptions)
                 {
                     _logger.LogError($"{nameof(RunExportersAsync)} failed. Message: {innerException.Message}");
                 }
-
-                IsSuccessful.Set(0);
             }
             catch (Exception e)
             {
                 _logger.LogError($"{nameof(RunExportersAsync)} failed. Message: {e.Message}");
```

Comment:
missing return (to avoid the scrap time logging)

Reply:
Added a return statement